### ISSUE TITLE
Configure asterisk alignment in multiline comments

### ISSUE THREAD

**Issue by normenmueller** [derekwyatt/vim-scala#116]
Any chance to configure `vim-scala` to have multiline docs aligned like suggested at http://goo.gl/7JxX2R.

For example, instead of 

```scala
/**
 * This is a multiline comment.
 *
 * Some more notes...
 */
```

rather get

```scala
/**
  * This is a multiline comment.
  *
  * Some more notes...
  */
```

**Comment by ches** [comment]
This is not something that's currently configurable. I'm sure we'd consider a patch to support it as an option since it is technically the Scaladoc recommended style, although I almost never see any projects in the wild use it other than the standard library…

**Comment by derekwyatt** [comment]
Yeah, personally, I much prefer it the way it is, and it would seem that most people agree.  However, as @ches said, we'd take a PR to make it configurable.  It always baffles me why people want to lose a column of usable space, but to each his own :)

**Comment by normenmueller** [comment]
@ches and @derekwyatt I'm willing to beta test! But as I already stated at https://github.com/derekwyatt/vim-scala/issues/117#issuecomment-166421772, I'm only a "Vim user" not used to "Vim programming".

**Comment by mjp4** [comment]
I would like to see this being added.  I have had a pull request with a fix waiting for a couple of months without any progress - is there something else I should do?  I have corrected a slight bug recently, but have been using it successfully.

**Comment by ches** [comment]
This was closed by #129, let us know if any issue arises. Thanks @mjp4!
